frameworks/helloworld/src/main/dist/svc.yml

```diff
@@ -9,11 +9,11 @@ pods:
     tasks:
       server:
         goal: RUNNING
-        cmd: "echo hello >> hello-container-path/output && sleep $SLEEP_DURATION"
+        cmd: "echo hello-$POD_INSTANCE_INDEX >> $OUTPUT_VOLUME/$OUTPUT_FILENAME && sleep $SLEEP_DURATION"
```

Comment:
I think it'd make sense to just put this in a separate yml that's exercised by an integration test. The `TASKCFG_*` stuff is only really useful in specific cases, and for newcomers it'd likely be confusing to have both env methods in the same place.